costmap_2d/plugins/static_layer.cpp

```diff
@@ -278,15 +278,30 @@ void StaticLayer::updateBounds(double robot_x, double robot_y, double robot_yaw,
 
   useExtraBounds(min_x, min_y, max_x, max_y);
 
-  double wx, wy;
+  geometry_msgs::TransformStamped transform;
+  try
+  {
+    transform = tf_->lookupTransform(global_frame_, map_frame_, ros::Time(0));
+  }
+  catch (tf2::TransformException ex)
+  {
+    ROS_ERROR("%s", ex.what());
```

Comment:
S: add info where the error is e.g. "static layer update bounds fails because %s"

Reply:
Hm I just copied the code from further below. But it's a TF problem, so the place where it happens is kind of irrelevant -- the error message is going to be self-explanatory "tf target frame failed blah blah". I think this is fine :shrug: